src/containers/VerifyActivities.jsx

```diff
@@ -2,6 +2,15 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
+// helpers
+import {
+  hasAllowedRole,
+  dateFormatter,
+  filterActivities,
+  filterActivitiesByStatus,
+} from '../helpers';
+import statsGenerator from '../helpers/statsGenerator';
```

Comment:
You can add `statsGenerator` to the index.js file in the helpers dir and import it with the other helpers